src/seedu/addressbook/AddressBook.java

```diff
@@ -574,17 +613,11 @@ private static String executeClearAddressBook() {
      * @return feedback display message for the operation result
      */
     private static String executeListAllPersonsInAddressBook() {
-        ArrayList<String[]> toBeDisplayed = getAllPersonsInAddressBook();
-        showToUser(toBeDisplayed);
+        ArrayList<HashMap<PersonProperty,String>> toBeDisplayed = getAllPersonsInAddressBook();
+        showPersonToUser(toBeDisplayed);
```

Comment:
by renaming `showToUser` to `showPersonToUser` you are reducing the generalization of the method; how?